source/python-sdk/structure/custom-harness.md

```diff
@@ -0,0 +1,38 @@
+# Harnesses
+
+While Vijil has a variety of pre-configured [harnesses](../../components/harnesses.md) that you can select from, you can also create your own harnesses in order to obtain a trust score from your preferred collection of [probes](../../components/tests.md).
+
+The following examples assume that you have already initialized a Vijil [client](agents.md).
+
+## Create harness
+
+Use the `Harnesses.create` function in the Python [client](agents.md) to specify the list of probes you want to include in the harness:
+
+```python
+client.Harnesses.create(name="my custom harness", probes=["probe1", "probe2"])
```

Comment:
let's do `client.harnesses.*` and `client.evaluations.*`
I'll roll in changes in the python client